src/app/util/attribute-storage.cpp

```diff
@@ -292,6 +293,29 @@ CHIP_ERROR emberAfSetDynamicEndpoint(uint16_t index, EndpointId id, const EmberA
         }
     }
 
+    size_t bufferSize = Compatibility::Internal::gEmberAttributeIOBufferSpan.size();
+    for (uint8_t i = 0; i < ep->clusterCount; i++)
+    {
+        const EmberAfCluster * cluster = &(ep->cluster[i]);
```

Comment:
Could also do:
```suggestion
        const EmberAfCluster & cluster = ep->cluster[i];
```
but either way.

Reply:
I'm going to leave the pointer-style since the structure is ultimately storing pointers -- in hindsight I probably should have done ep->cluster + i to just access the pointer directly.

Reply:
For what it's worth, the structure is not in fact storing pointers.  It's storing EmberAfCluster objects.

Reply:
@witowsg-amazon